README.md

```diff
@@ -232,6 +232,7 @@ where:
 - `opts` *Object*:
   - `answers` *Array*: A collection of questions and their answers of form `[{ q: String, a: String }, ...]`
   - `comment` *String*: A text to be viewed by the Kaitiakis of a group
+  - `profileId` *String*: The msgId of the profile holding the applicants information
```

Comment:
might want to say "Note, if this profile is encrypted, it should be encrypted so `groupAdmins` can read it"